packages/docs/modules/page-config/blocks/change-log/index.ts

```diff
@@ -0,0 +1,14 @@
+import { definePageConfigBlock } from '../../types'
```

Comment:
This example doesn't allow me to type. Letter appears and instantly disappears.

<img width="492" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/5394573/dcad02b2-c004-4076-a288-dbc136da75c1">

Hard to test further before that's fixed.

Reply:
Must be fixed in https://github.com/epicmaxco/vuestic-ui/pull/3642/commits/937bedf0d1221278dffe1977cd3aa7bab2457b23.